src/app/components/request-storage/request-storage.component.ts

```diff
@@ -321,59 +372,64 @@ export class RequestStorageComponent implements OnInit, OnDestroy, CanComponentD
     if (form !== null) {
       form = JSON.parse(form);
 
-      this.requestTypeForm.setValue(form['requestTypeForm']);
-      this.requestDetailsForm.setValue(form['requestDetailsForm']);
-      this.projectForm.setValue(form['projectForm']);
-      this.dataInfoForm.setValue(form['dataInfoForm']);
-      this.dataSizeForm.setValue(form['dataSizeForm']);
+      this.requestTypeForm.setValue(form["requestTypeForm"]);
+      this.requestDetailsForm.setValue(form["requestDetailsForm"]);
+      this.projectForm.setValue(form["projectForm"]);
+      this.dataInfoForm.setValue(form["dataInfoForm"]);
+      this.dataSizeForm.setValue(form["dataSizeForm"]);
 
       this.stepper.selectedIndex = this.stepper._steps.length - 1; // Navigate to last step
     }
   }
 
   clearRequest() {
-    localStorage.removeItem(this.requestFormKey)
+    localStorage.removeItem(this.requestFormKey);
+  }
+
+  /**
+   * Unchecks the personal dropbox checkbox on the first page.
+   */
+  uncheckPersonalDropboxCheckbox() {
+    this.requestTypeForm.controls.isPersonalDropbox.setValue(false);
   }
 
   addNewPerson() {
     this.addPerson({
       firstName: undefined,
       lastName: undefined,
       email: undefined,
-      access: 'Full Access',
+      access: "Full Access",
       roles: {
         dataOwner: false,
         dataContact: false,
         projectOwner: false,
-      }
+      },
     });
   }
 
   addPerson(person: Person) {
-    const control = <FormArray>this.dataInfoForm.get('projectMembers');
+    const control = <FormArray>this.dataInfoForm.get("projectMembers");
     const rolesFormGroup = this.formBuilder.group({
       dataContact: new FormControl(person.roles.dataContact),
       projectOwner: new FormControl(person.roles.projectOwner),
-      dataOwner: new FormControl(person.roles.dataOwner)
+      dataOwner: new FormControl(person.roles.dataOwner),
     });
 
     const personFormGroup = this.formBuilder.group({
       firstName: new FormControl(person.firstName, Validators.required),
       lastName: new FormControl(person.lastName, Validators.required),
       email: new FormControl(person.email, [
         Validators.required,
-        Validators.pattern('.*(aucklanduni.ac.nz|auckland.ac.nz)$')
+        Validators.pattern(/.*(?<=\@|\.)(aucklanduni.ac.nz|auckland.ac.nz)$/),
```

Comment:
Got a report saying the Hub's storage request feature isn't working in Safari. Investigated and found Safari does not support this regular expression. This error is given:
`SyntaxError: Invalid regular expression: invalid group specifier name` :thinking:

Reply:
Looks like the regex lookbehind syntax isn't supported by Safari yet :disappointed: https://caniuse.com/js-regexp-lookbehind